src/main/groovy/life/qbic/datamodel/accounting/CostEstimate.groovy

```diff
@@ -19,12 +21,22 @@ class CostEstimate {
     /**
      * The date on which the cost estimate was created
      */
-    final Date creationDate
+    final Date modificationDate
+
+    /**
+     * The date on which the cost estimate expires
+     */
+    final Date expirationDate
 
     /**
      * The customer for which the estimate is created
      */
-    final Person customer
+    final Customer customer
+
+    /**
+     * The QBiC project manager who was assigned to the project
+     */
+    final Person projectManager
```

Comment:
In which order do you plan to merge the PRs of yours. The one introducing the projectmanager object should be either way reflected here at some point. In this PR or in the other one so that we won't miss it :)

Reply:
Good Question, I was unsure how to proceed here since the projectManager Class is in a seperate PR and i wanted to make sure this PR works properly in it's current state. 
I'd rather merge the other pr into development and merge that back into this PR and adapt the property to make sure everything runs properly.

Reply:
Thats a good idea, then I will wait for your merge until my final approval :)